CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Resources/en.lproj/Localizable.strings

```diff
@@ -174,7 +174,7 @@ conveniently drag-and-drop PDFs from the iOS Files app to your Banking App to st
 "ginicapture.error.connection.title" = "There was a problem connecting to the internet";
 "ginicapture.error.authentication.title" = "There has been an authentication error";
 "ginicapture.error.serverError.title" = "Service unavailable";
-"ginicapture.error.unexpected.title" = "Oh something went wrong";
+"ginicapture.error.unexpected.title" = "An unknown error occured.";
```

Comment:
[Request] Please also update the German translation: https://www.figma.com/file/m4OtCvxWxPMeB0Z5k4Aeow/Gini-Photo-Payment-%2F-iOS?node-id=4845%3A63183&t=WNKUNqRB9t2gVWQ4-0

Reply:
Oh and "occured" is spelled incorrectly: it should be "occurred" -> `An unknown error occurred.`